messages/marshal.go

```diff
@@ -0,0 +1,16 @@
+package messages
+
+import "time"
+
+// businessDateFormat is the format used for business dates in messages.
+const businessDateFormat = "2006-01-02"
+
+// MarshalDate marshals a time value to a date string.
```

Comment:
Has this not been rebased off `master`. I thought these functions where gone?

Reply:
It had been. Maybe they got re-added during the merge.

Reply:
Actually these seem like they are still needed for validation. I could move them into the `internal/validation/validation.go` file though?

Reply:
Are they really both needed?

I think you could just make `IsValidBusinessDate()` do what it needs to do.

Reply:
The `marshal.go` file has been removed and `validation.go` updated to do all it needs to validate a date.